app/models/rss_log.rb

```diff
@@ -437,7 +437,7 @@ def decode_args(args)
   end
 
   def decode_time(str)
-    Time.parse(str).in_time_zone
+    Time.parse(str + "+0000").in_time_zone
```

Comment:
FWIW, Rubocop will almost certainly prefer "#{str}+0000".

Reply:
I would have agreed with you, but I just checked and it doesn't seem to care.  Neither way generates a violation and it's exactly the same number of characters.  For some reason I don't fully understand I slight prefer the way I wrote it, but like Rubocop, I don't have strong opinion. :-)